packages/layouts-react/src/components/PageLayout/PageLayout.tsx

```diff
@@ -33,6 +34,7 @@ PageLayout.SideNavigation = SideNavigation;
 PageLayout.Content = Content;
 PageLayout.ToolsArea = ToolsArea;
 PageLayout.TitleArea = TitleArea;
+PageLayout.BottomBar = BottomBar;
 PageLayout.ActionBar = ActionBar;
```

Comment:
I have moved `ActionBar` into its own folder under `components`. I think the css remains the same? Since it is dependent on the demo layout which has `ActionBar` under `BottomBar` which is subcomponent of `PageLayout`.

Reply:
I'd like to remove action bar from under bottom bar (at least).
Also BottomBar doesn't need background - leave it completely empty, just height and area in the grid.

Reply:
I'm still working on adjusting the css for this because moving it out is messing up the whole interface that previously relied on being under `BottomBar` and `PageLayout`. But just to make absolutely sure I understand correctly, do you mean for ActionBar to be placed like this?
![image](https://user-images.githubusercontent.com/22462084/234713430-4f553bf7-889b-4351-8233-c1ffc4c7d720.png)
and still rely on page layout grid to position at the bottom.

Or like this:
![image](https://user-images.githubusercontent.com/22462084/234713727-0507f252-b7ed-45c7-a394-005b80f775a3.png)
Where it is completely outside of PageLayout and BottomBar, and will have independent positioning (to be fixed at the bottom)? and its own styling.

Reply:
Looking back to my initial examples: https://github.com/iTwin/iTwinUI-layouts/pull/70#issuecomment-1517594060

```ts
<PageLayout>
  <PageLayout.Content>
    {content}
  </PageLayout.Content>
  <PageLayout.BottomBar>
    {bottom bar content, might be ActionBar}
  </PageLayout.BottomBar>
</PageLayout>
```

Reply:
CSS should be something like:
```css
.iui-layouts-page {
  &-bottom-bar {
    grid-area: bottom-bar;
    height: calc(var(--iui-size-s)*5 + var(--iui-size-l)); //this is the sizes of the two bottom buttons of side nav
  }

  &-action-bar {
    [action bar css code]
  }
}
```

or 
```css
.iui-layouts-page {
  &-bottom-bar {
    grid-area: bottom-bar;
    height: calc(var(--iui-size-s)*5 + var(--iui-size-l)); //this is the sizes of the two bottom buttons of side nav
  }
}

.iui-action-bar {
  [action bar css code]
}
```

Reply:
Okay I updated this.

Reply:
Did you push these changes?I I'm not seeing it.

Reply:
My bad, it should show now.